packages/gatsby-recipes/recipes/mdx-pages.mdx

```diff
@@ -0,0 +1,68 @@
+# Setup MDX pages within Gatsby
+
+MDX is markdown for the component era.
+
+Write markdown with embedded React components.
+Awesome 🤯
+
+This recipe installs and configures all
+the necessary NPM packages & Gatsby Plugins
+to enable auto-creation of MDX pages within
+src/pages.
+
+---
+
+Install NPM Packages
+
+<NPMPackage
+  name="gatsby-source-filesystem"
+/>
+<NPMPackage
+  name="gatsby-plugin-mdx"
+/>
+<NPMPackage
+  name="@mdx-js/mdx"
+/>
+<NPMPackage
+  name="@mdx-js/react"
+/>
+
+---
+
+Install the Gatsby Plugins.
+
+First `gatsby-source-wordpress` to read the MDX files in the src/pages
+directory.
+
+<GatsbyPlugin
+  name="gatsby-source-filesystem"
+  key="src/pages"
+  options={{
+    name: `MDXPages`,
+    path: `src/pages/`,
+  }}
+/>
+
+---
+
+Then `gatsby-plugin-mdx` to convert MDX files to pages.
```

Comment:
An empty line isn't exactly required, but maybe we should break up the command from its description?

```suggestion
Then `gatsby-plugin-mdx` to convert MDX files to pages.

```